manuscripts2/new_functions.py

```diff
@@ -0,0 +1,510 @@
+import logging
```

Comment:
You need to add the license/copyright template. Please, get it from some other file, and include it here.